examples/webgpu_loader_gltf.html

```diff
@@ -65,61 +64,45 @@
 
 				scene = new THREE.Scene();
 
-				const rgbmUrls = [ 'px.png', 'nx.png', 'py.png', 'ny.png', 'pz.png', 'nz.png' ];
+				new RGBELoader()
+					.setPath( 'textures/equirectangular/' )
+					.load( 'royal_esplanade_1k.hdr', function ( texture ) {
 
-				const cubeMap = new RGBMLoader()
-					.setMaxRange( 16 )
-					.setPath( './textures/cube/pisaRGBM16/' )
-					.loadCubemap( rgbmUrls );
+						texture.mapping = THREE.EquirectangularReflectionMapping;
 
-				cubeMap.generateMipmaps = true;
-				cubeMap.minFilter = THREE.LinearMipmapLinearFilter;
+						scene.background = texture;
+						scene.environment = texture;
 
-				scene.environmentNode = cubeTexture( cubeMap );
-				scene.backgroundNode = scene.environmentNode;
+						render();
 
-				const loader = new GLTFLoader().setPath( 'models/gltf/DamagedHelmet/glTF/' );
-				loader.load( 'DamagedHelmet.gltf', function ( gltf ) {
+						// model
 
-					//const light = new THREE.PointLight( 0xffffff );
-					//camera.add( light );
+						const loader = new GLTFLoader().setPath( 'models/gltf/DamagedHelmet/glTF/' );
+						loader.load( 'DamagedHelmet.gltf', function ( gltf ) {
 
-					const mesh = gltf.scene.children[ 0 ];
-					const material = mesh.material;
+							scene.add( gltf.scene );
```

Comment:
Why did you remove `normalMap` setting's lines?

Reply:
I just copy/pasted the code from `webgl_loader_gltf`.